doc/SLSKPROTOCOL.md

```diff
@@ -2289,22 +2289,25 @@ a peer. In Nicotine+, these messages are defined in slskmessages.py.
 (slskd, Seeker)*
 
 1.  User A sends [ConnectToPeer](#server-code-18) to the Server with a unique
-    token (indirect connection request)
-2.  User A sends a [PeerInit](#peer-init-code-1) to User B (direct connection
-    request)
-3.  The Server sends a [ConnectToPeer](#server-code-18) response to User B with
-    the same token.  
+    token (indirect connection request) and User B's username. The token is stored
+    for later use (see steps 4 and 5).
+2.  User A sends [GetPeerAddress](#server-code-3) to the Server with User B's username.
+3.  The Server responds to User A with [GetPeerAddress](#server-code-3), which gives User A the IP of User B.
+4.  User A sends a [PeerInit](#peer-init-code-1) to User B (direct connection
```

Comment:
I didn't catch this before merging, but the token in the PeerInit is always 0, and goes unused (probably a leftover from the early days of the Soulseek protocol).